docs/reference/motlp.md

```diff
@@ -103,8 +121,31 @@ If send data that exceeds the available rate limit, the {{motlp}} will respond w
 }
 ```
 
-Once your sending rate drops back within the allowed limit, the system will automatically begin accepting requests again.
+Once your sending rate drops back within the allowed limit, the system automatically begins accepting requests again.
 
 :::{note}
 If you need to increase the rate limit, reach out to Elastic Support.
-:::
+::: get rate limiting errors. To solve this issue, scale or resize your deployment:
+
+- [Scaling considerations](docs-content://deploy-manage/production-guidance/scaling-considerations.md)
+- [Resize deployment](docs-content://deploy-manage/deploy/cloud-enterprise/resize-deployment.md)
+- [Autoscaling in ECE and ECH](docs-content://deploy-manage/autoscaling/autoscaling-in-ece-and-ech.md)
+
+### Exceeding the rate limit
+
+The rate limit is currently set to 15 MB/s per second, with a burst limit of 30 MB/s per second. As long as your data ingestion rate stays at or below this average, your requests will be accepted.
```

Comment:
How about listing the concrete numbers in the section above and focusing here on 
- serverless: decrease data or reach out 
- deployment: what is described in [L128:132](https://github.com/elastic/opentelemetry/pull/443/files#diff-d02fc62bb3da13f6b80846b87b3033b88919b735fe518cd6339380a27160f18eR128-R132)